maliput_sim/core/components.py

```diff
@@ -120,3 +140,8 @@ class RoadNetwork(Component):
     def __init__(self, road_network):
         super().__init__()
         self.road_network = road_network
+
+    def get_state(self):
+        # TODO: Evaluate if this is the best way to store the maliput road network. RoadNetwork is stateless
+        #       and probably doesn't need to be stored in the ECM.
```

Comment:
```suggestion
        # TODO: Evaluate if this is the best way to store the maliput road network.
```

It is not stateless (think of the controllers for the phases, state providers in general). However, I see the point of not returning anything.

Reply:
Good point